02.calendar/cal.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+import dayjs from "dayjs";
+import ja from "dayjs/locale/ja.js";
+
+dayjs.locale(ja);
+const argv = minimist(process.argv.slice(2));
+
+let month = dayjs().format("M");
+let year = dayjs().format("YYYY");
+
+if (argv["m"]) {
+  month = String(argv["m"]);
+  if (argv["y"]) {
+    year = String(argv["y"]);
+  }
+}
+
+const formatted_month_and_year = `${year}-${month}`;
```

Comment:
```suggestion
const formattedMonthAndYear = `${year}-${month}`;
```

JSは基本的にキャメルケースです

あと変数名がちょっと長いので`yearMonth`ぐらいでもいいかもしれません〜